buff.lic

```diff
@@ -21,6 +21,10 @@ class Waggle
 
     waggle_spells = @settings.waggle_sets
 
+    if args.spells['prehunt_buffs']
+      return unless waggle_spells[args.spells]
+    end
+
```

Comment:
I'd rather not have a specific check here, `buff.lic` shouldn't be concerned with which specific set it was told to do.

I don't think it defaults the way you're suggesting. `Hunting-buddy` will be passing in an arg all the time, so `args.spells` will be whatever you pass in. Then `waggle_spells[args.spells]` will resolve, I assume to `nil` (that might be worth double checking). That is what we would want to check for, perhaps `return unless spells` in `check_spells(...)`?